### PR TITLE
#162633452 Set up redux

### PR DESCRIPTION
**Pull request by fatukunda**
### What does this PR do?

Setup Redux in the application

**Description of Tasks to be completed?**

 - After cloning the repo, `cd` into it and run `yarn install` to install dependencies.
 - Install the redux dev tools extension in your browser.

**How should this be manually tested?**

- Run yarn start to run the application.

- Check your redux dev tools to confirm that the state in your app appears in the dev tools.

**What are the relevant pivotal tracker stories?**

[#162633452](https://www.pivotaltracker.com/story/show/162633452)

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//Sample actions
+export const FETCH_ARTICLES_SUCCESS = "FETCH_ARTICLES_SUCCESS";
+
```
Comment (mubstimor) [inline src/actions/index.js:5]: why is there a blank line here?

**Comment by araaliFarooq** [comment]
LGTM

File: src/reducers/articlesReducer.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+//A test article reducer. This can be deleted and replaced with a real article reducer
+import { FETCH_ARTICLES_SUCCESS } from "../actions/index";
```
[the remark this answers is not in the document]
Comment (mubstimor) [reply src/reducers/articlesReducer.js]: please address these hound violations.

File: src/actions/index.js (hunk 1 above)
Comment (fatukunda) [reply src/actions/index.js:5]: @mubstimor That is to separate the variable declaration from the method.

**Comment by NabunyaLilian** [comment]
Good work @fatukunda , though could remove the quotes on the link in the PR description and add square brackets ([]) on the story id to make your description more appealing and accessible.

**Comment by mubstimor** [comment]
please address these hound violations first

**Comment by Qadriyah** [comment]
LGTM

**Review by ruganda — APPROVED** [review]

**Pull request merged**